### ISSUE TITLE
requirements+python

### ISSUE THREAD

**Issue by Madifaye** [wangpei066-hue/CoreGeek#1343]
Python 3.11.10
absl-py==2.1.0
astunparse==1.6.3
blinker==1.8.2
cachetools==5.5.0
certifi==2024.8.30
charset-normalizer==3.3.2
click==8.1.7
filelock==3.16.0
Flask==3.0.3
flatbuffers==24.3.25
fsspec==2024.9.0
gast==0.6.0
google-auth==2.34.0
google-pasta==0.2.0
grpcio==1.66.1
h5py==3.11.0
idna==3.8
itsdangerous==2.2.0
Jinja2==3.1.4
keras==3.5.0
Keras-Preprocessing==1.1.2
libclang==18.1.1
Markdown==3.7
markdown-it-py==3.0.0
MarkupSafe==2.1.5
mdurl==0.1.2
ml-dtypes==0.4.0
mpmath==1.3.0
namex==0.0.8
networkx==3.3
numpy==1.26.4
opencv-python==4.10.0.84
opt-einsum==3.3.0
optree==0.12.1
packaging==24.1
pandas==2.2.2
pillow==10.4.0
protobuf==4.25.4
pyasn1==0.6.0
pyasn1_modules==0.4.0
Pygments==2.18.0
python-dateutil==2.9.0.post0
pytz==2024.1
PyYAML==6.0.2
requests==2.32.3
rich==13.8.0
rsa==4.9
six==1.16.0
sympy==1.13.2
tensorboard==2.17.1
tensorboard-data-server==0.7.2
tensorflow==2.17.0
tensorflow-io-gcs-filesystem==0.37.1
termcolor==2.4.0
torch==2.4.1
torchvision==0.19.1
triton==3.0.0
typing_extensions==4.12.2
tzdata==2024.1
urllib3==2.2.2
Werkzeug==3.0.4
wrapt==1.16.0
